src/index.html

```diff
@@ -17,11 +17,256 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`styles/main.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS into CSS and link to the resulting CSS file instead.